unstructured/partition/pdf.py

```diff
@@ -650,3 +698,44 @@ def _partition_pdf_or_image_with_ocr(
             if include_page_breaks:
                 elements.append(PageBreak(text=""))
     return elements
+
+
+def check_coords_within_boundary(
```

Comment:
Oh, I created a similar function in `unstructured_inference`, probably can be unified at some point.

Reply:
I'll have a note on this!